### ISSUE TITLE
Support Flighting

### ISSUE THREAD

**Issue by gaufung** [AiursoftWeb/Infrastructures#249]

**Comment by Anduin2017** [comment]
No. Flighting doesn't mean live configuration outside Microsoft. So please do not mention  'flight' as live configuration here.

**Comment by gaufung** [comment]
> No. Flighting doesn't mean live configuration outside Microsoft. So please do not mention 'flight' as live configuration here.

I don't think so. App Store has an application called **TestFlight**

**Comment by Anduin2017** [comment]
Just searched on the web. The word `flight` has no meaning or any relationship between `live configuration`.

![image](https://user-images.githubusercontent.com/19531547/113551264-f3509400-9626-11eb-9376-d148b5596877.png)

**Comment by gaufung** [comment]
> Just searched on the web. The word `flight` has no meaning or any relationship between `live configuration`.
> 
> ![image](https://user-images.githubusercontent.com/19531547/113551264-f3509400-9626-11eb-9376-d148b5596877.png)

Looks interesting 
https://aidanfinn.com/?p=17862
